apps/docs/src/app/core/component-docs/carousel/examples/carousel-content-navigation.component.html

```diff
@@ -0,0 +1,39 @@
+<fd-carousel [navigatorInPageIndicator]="false" [pageIndicatorContainerPlacement]="'top'">
```

Comment:
remove `"'top'"` [] binding here and at other places

Reply:
fixed